src/test/java/org/prebid/server/bidder/vox/VoxBidderTest.java

```diff
@@ -0,0 +1,178 @@
+
+        final BidderCall<BidRequest> httpCall = givenHttpCall(
+                givenBidRequest(),
+                givenBidResponse(bannerBid, videoBid, audioBid, nativeBid));
+
+        // when
+        final Result<List<BidderBid>> result = target.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).containsExactly(
+                BidderBid.of(bannerBid, banner, "USD"),
+                BidderBid.of(videoBid, video, "USD"),
+                BidderBid.of(audioBid, audio, "USD"),
+                BidderBid.of(nativeBid, xNative, "USD"));
+    }
+
+    @Test
+    public void makeBidsShouldReturnErrorWhenImpTypeIsNotSupported() throws JsonProcessingException {
+        // given
+        final Bid audioBid = Bid.builder().impid("id").mtype(5).build();
+        final BidderCall<BidRequest> httpCall = givenHttpCall(
+                givenBidRequest(),
+                givenBidResponse(audioBid));
+
+        // when
+        final Result<List<BidderBid>> result = target.makeBids(httpCall, null);
+        // then
+        assertThat(result.getErrors())
+                .containsExactly(badServerResponse("Unable to fetch mediaType 5 in multi-format: id"));
+    }
+
+    @Test
+    public void makeBidsShouldReturnErrorWhenResponseDoesNotHaveSeatBid() throws JsonProcessingException {
+        // given
+        final BidderCall<BidRequest> httpCall = givenHttpCall(null, givenBidResponse());
+
+        // when
+        final Result<List<BidderBid>> actual = target.makeBids(httpCall, null);
+
+        // then
+        assertThat(actual.getValue()).isEmpty();
+        assertThat(actual.getErrors()).containsExactly(badServerResponse("Empty SeatBid array"));
+    }
+
+    private static BidRequest givenBidRequest() {
+        final Imp imp = Imp.builder()
+                .id("imp_id")
+                .ext(mapper.valueToTree(ExtPrebid.of(
+                        null,
+                        new VoxImpExt("placementId", "http://some.image.url", List.of("1920x1080")))))
+                .build();
+        return BidRequest.builder().imp(List.of(imp)).build();
+    }
+
+    private static String givenBidResponse(Bid... bids) throws JsonProcessingException {
+        return mapper.writeValueAsString(
+                BidResponse.builder()
+                        .cur("USD")
+                        .seatbid(bids.length == 0
+                                ? Collections.emptyList()
+                                : List.of(SeatBid.builder().bid(List.of(bids)).build()))
+                        .build());
+    }
+
+    private static BidderCall<BidRequest> givenHttpCall(BidRequest bidRequest, String body) {
+        return BidderCall.succeededHttp(
+                HttpRequest.<BidRequest>builder().payload(bidRequest).build(),
+                HttpResponse.of(200, null, body),
+                null);
+    }
+
+    @Value
```

Comment:
@Value(staticConstructor = "of") than replace `new` creation approach with static method